docs/admin/install.md

```diff
@@ -19,7 +22,13 @@ The plugin is available as a Python package via PyPI and can be installed with `
 pip install nautobot-ssot
 ```
 
-To ensure Single Source of Truth is automatically re-installed during future upgrades, create a file named `local_requirements.txt` (if not already existing) in the Nautobot root directory (alongside `requirements.txt`) and list the `nautobot-ssot` package:
+To use specific integrations, add them as extra dependencies:
+
+```shell
+pip install nautobot-ssot[aci]
```

Comment:
I would show an example with at least two integrations to show the syntax

Reply:
Fixed